Tests/test_bgzf.py

```diff
@@ -128,9 +128,9 @@ def check_by_line(self, old_file, new_file, old_gzip=False):
             # Seems gzip can return bytes even if mode="r",
             # perhaps a bug in Python 3.2?
```

Comment:
This is intended behavior, not a bug in Python.

Reply:
Bug vs feature, Python 2 vs Python 3 defaults were unfortunate here.

Reply:
That may be, but suggesting that this perhaps is a bug points people in the wrong direction, and is insulting to the programming language that we hold dear.

Reply:
Teething issues and bugs like https://bugs.python.org/issue13989 were to be expected in the early Python 3 releases - would you like to suggest a better wording now that we're past that, and require Python 3.6+ for Biopython?

In any case, the comment wording is not directly relevant to this pull request.

Reply:
Just remove it.

Reply:
OK 👍